docs/DEVELOPER.md

```diff
@@ -0,0 +1,37 @@
+# Instructions for packaging ConStrain package locally with Poetry
+
+ConStrain can be made a pip-able Python package using `poetry`. Here's a quick guide to go about creating the package, installing it, and testing it.
+
+## Install `poetry`
+```
+curl -sSL https://install.python-poetry.org | python3 -
```

Comment:
Should we also add instructions for Windows? Or at least specify that this is for Unix?